src/Core/Services/Implementations/UserService.cs

```diff
@@ -127,6 +127,10 @@ public class UserService : UserManager<User>, IUserService, IDisposable
 
             return userIdGuid;
         }
+        public override string GetUserName(ClaimsPrincipal principal)
```

Comment:
Is this override necessary? If the base class already implements this member, the interface should be satisfied as-is w/o having to override and call `base`.

Reply:
You're right, good catch. Removed.